lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -2979,7 +2978,7 @@ namespace Js
 
     bool JavascriptLibrary::InitializeDatePrototype(DynamicObject* datePrototype, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode)
     {
-        typeHandler->Convert(datePrototype, mode, 51);
+        typeHandler->Convert(datePrototype, mode, 48);
```

Comment:
Expected to decrease this by 1 (to avoid allocating space for a pointer to getVarDate) but counted all of the members of the datePrototype and found only 48 not 50 so dropped to 48.